test/integration/targets/nxos_bgp/tests/cli/sanity.yaml

```diff
@@ -0,0 +1,132 @@
+---
+- debug: msg="START TRANSPORT:CLI nxos_bgp sanity test"
+
+- set_fact: suppress_fib_pending="true"
+  when: (titanium is defined) and ((titanium | search('true')) != 'true')
```

Comment:
This test did not work properly as written. `when: (titanium is defined) and not ((titanium | search('true')))` did.